airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicFunctionalityIntegrationTest.kt

```diff
@@ -397,6 +397,7 @@ abstract class BasicFunctionalityIntegrationTest(
     }
 
     @Test
+    @Disabled
```

Comment:
I disabled this because of https://scans.gradle.com/s/qj6xwb73ceh5k/tests/task/:airbyte-cdk:bulk:core:bulk-cdk-core-load:integrationTest/details/io.airbyte.cdk.load.mock_integration_test.MockBasicFunctionalityIntegrationTest/testMidSyncCheckpointingStreamState()?top-execution=1